addons/skin.estuary/xml/DialogAddonSettings.xml

```diff
@@ -123,6 +123,53 @@
 					<param name="label" value="" />
 				</include>
 			</control>
+			<control type="group">
+				<description>Currently-playing emulator name and icon</description>
+				<visible>Player.HasGame + String.IsEqual(Window(addonsettings).Property(Addon.Type),kodi.gameclient)</visible>
+				<left>1480</left>
+				<width>310</width>
+				<top>382</top>
+				<height>330</height>
+				<control type="group">
+					<left>52</left>
+					<control type="image">
+						<description>Background image</description>
+						<left>-20</left>
+						<right>-20</right>
+						<top>-20</top>
+						<bottom>-20</bottom>
+						<texture border="40">buttons/dialogbutton-nofo.png</texture>
+					</control>
+					<control type="label">
+						<description>Label for "Playing with:" text</description>
+						<top>14</top>
+						<height>20</height>
+						<font>font16</font>
+						<shadowcolor>text_shadow</shadowcolor>
+						<label>35271</label>
```

Comment:
I'm not sure it really matters, but the usual convention would be `$LOCALIZE[35271]`

Reply:
Yeah, probably a good idea to with convention for the default skin.

Reply:
It doesn't matter from a skin perspective, but better to be explicit and follow convention.